apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/CloudTrace/RateLimitingTraceOptionsFactoryTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Moq;
+using Xunit;
+
+namespace Google.Devtools.AspNet.Tests.CloudTrace
+{
+    public class RateLimitingTraceOptionsFactoryTest
+    {
+        // Creates a rate limiter that will allow 1 QPS with the elapsed milliseconds.
+        private RateLimiter GetRateLimiter(long elapsedMilliseconds)
```

Comment:
This looks like it would also be useful in RateLimiterTest. Perhaps move to a static utilities class.